fastlane/Fastfile

```diff
@@ -150,6 +154,33 @@ lane(:generate_swift_api) do |options|
   generated_files
 end
 
+desc("Generate the FastlaneSwiftRunner project manifest")
+lane(:generate_fastlane_swift_runner_manifest) do |options|
+  # Don't attempt to upgrade config files, like Fastfile.swift, Deliverfile.swift, etc...
+  non_upgrade_set = TOOL_CONFIG_FILES.map { |tool_name| "#{tool_name}.swift" }.to_set
+
+  swift_folder = File.expand_path("swift")
+  runner_project_path = File.expand_path(File.join(swift_folder, "/FastlaneSwiftRunner/FastlaneSwiftRunner.xcodeproj"))
+  project = Xcodeproj::Project.open(runner_project_path)
+  swift_files = project.files.select do |file|
```

Comment:
Maybe this piece is easier to read if we have multiple `.select do` blocks? It would take longer to run, but this should be done in less than a second anyway, right?